modules/machine-health-checks-resource.adoc

```diff
@@ -17,27 +17,30 @@ kind: MachineHealthCheck
 metadata:
   name: example <1>
   namespace: openshift-machine-api
+  annotations:
+     <2>
 spec:
   selector:
     matchLabels:
-      machine.openshift.io/cluster-api-machine-role: <role> <2>
-      machine.openshift.io/cluster-api-machine-type: <role> <2>
-      machine.openshift.io/cluster-api-machineset: <cluster_name>-<label>-<zone> <3>
+      machine.openshift.io/cluster-api-machine-role: <role> <3>
+      machine.openshift.io/cluster-api-machine-type: <role> <3>
+      machine.openshift.io/cluster-api-machineset: <cluster_name>-<label>-<zone> <4>
   unhealthyConditions:
   - type:    "Ready"
-    timeout: "300s" <4>
+    timeout: "300s" <5>
     status: "False"
   - type:    "Ready"
-    timeout: "300s" <4>
+    timeout: "300s" <5>
     status: "Unknown"
-  maxUnhealthy: "40%" <5>
+  maxUnhealthy: "40%" <6>
 ----
 <1> Specify the name of the MachineHealthCheck to deploy.
-<2> Specify a label for the machine pool that you want to check.
-<3> Specify the MachineSet to track in `<cluster_name>-<label>-<zone>`
+<2> Add `machine.openshift.io/remediation-strategy: external-baremetal` annotation to enable power-cycle remediation for bare metal.
```

Comment:
```suggestion
<2> For bare metal clusters, add the `machine.openshift.io/remediation-strategy: external-baremetal` annotation to the `matchLabels` section to enable power-cycle remediation. With this remediation strategy, unhealthy hosts are rebooted instead of removed from the cluster.
```

Reply:
Thanks @kalexand-rh 
I'm not sure I understand the last suggestions.

The annotation is not related to the `matchLabel` sections.
Here's a full example with that annotation, if it helps to clarify things:
```
apiVersion: machine.openshift.io/v1beta1
kind: MachineHealthCheck
metadata:
  name: example 
  namespace: openshift-machine-api
  annotations:
    machine.openshift.io/remediation-strategy: external-baremetal
spec:
  selector:
    matchLabels:
      machine.openshift.io/cluster-api-machine-role: <role> 
      machine.openshift.io/cluster-api-machine-type: <role> 
      machine.openshift.io/cluster-api-machineset: <cluster_name>-<label>-<zone> 
  unhealthyConditions:
  - type:    "Ready"
    timeout: "300s" 
    status: "False"
  - type:    "Ready"
    timeout: "300s" 
    status: "Unknown"
  maxUnhealthy: "40%" 
```

Reply:
@n1r1, thanks for the clarification. I've updated the suggestion.

Reply:
@kalexand-rh , thanks for the suggestion, it makes sense and I've now accepted it.
Please let me know if anything else is needed